test/asserts/check.go

```diff
@@ -0,0 +1,7 @@
+package asserts
+
+// transparently checks the error value returned by a fallible operation
+func Check1[T any](value T, err error) T { //nolint:ireturn
```

Comment:
This maybe?

```suggestion
func NoError1[T any](value T, err error) T { //nolint:ireturn
```

Reply:
I like it, thanks for the suggestion! :pray: